lib/multisigHelpers.ts

```diff
@@ -40,12 +46,13 @@ const createMultisigFromCompressedSecp256k1Pubkeys = async (
   return res.data.address;
 };
 
-/** Like Account but with non-optional pubkey */
-export type AccountWithPubkey = Account & { readonly pubkey: Pubkey };
-
 /**
  * This gets a multisigs account (pubkey, sequence, account number, etc) from
- * a node and/or the api if the multisig was made on this app
+ * a node and/or the api if the multisig was made on this app.
+ *
+ * The public key should always be available, either on chain or in the app's database.
+ * The account is only available when the there was any on-chain activity such as
+ * receipt of tokens.
  *
  * @param {string} address The multisig address
  * @param client A connected stargate cosmoshub client
```

Comment:
the documentation of the return value of this function should be updated to be an array. Im also ok with removing the `param` and `return` comments here and just relying on typescript